iabtcf-extras-jackson/src/main/java/com/iabtcf/extras/jackson/Cmp.java

```diff
@@ -0,0 +1,90 @@
+package com.iabtcf.extras.jackson;
+
+/*-
+ * #%L
+ * IAB TCF Java GVL and CMP List Jackson
+ * %%
+ * Copyright (C) 2020 IAB Technology Laboratory, Inc
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.time.Instant;
+import java.util.Optional;
+
+public class Cmp implements com.iabtcf.extras.Cmp {
+
+    private int id;
+    private String name;
+
+    @JsonProperty("isCommercial")
```

Comment:
why does this attribute need the JsonProperty tag?

Reply:
The field name in the JSON schema itself is IsCommercial. Unless I change my field name to isIsCommercial (two  "is"), the  jackson parser will not work. Hence I need to tell the jackson parser that the property name is isCommercial.

Reply:
that makes sense. thanks for explaining